src/ProgressBar/ProgressBar.stories.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import {Meta, ComponentStory} from '@storybook/react'
+import ProgressBar from './ProgressBar'
+
+export default {
+  title: 'Components/ProgressBar',
+  component: ProgressBar,
+} as Meta<typeof ProgressBar>
+
+export const Default = () => <ProgressBar />
+
+export const Playground: ComponentStory<typeof ProgressBar> = args => (
+  <ProgressBar {...args} sx={args.inline ? {width: '100px'} : {}} />
```

Comment:
When using `inline` [the docs mention](https://primer.style/react/ProgressBar#inline) to also set a `width`. Which I tried adding here. It works, but having the `width` set automatically could suggest that it's part of the component and people might get confused when not also reading the docs.

Is this a big concern? A better approach? Or should we remove the `inline` option again from the Playground to avoid confusion?

Reply:
Ok, I'll merge it as is. In the future we might could add a `min-width` so that manually setting a `width` is not necessarily needed.